templates/SilverStripe/Forms/GridField/GridFieldViewButton.ss

```diff
@@ -1 +1 @@
-<a class="action action-detail view-link" href="$Link">View</a>
+<a class="grid-field__icon-action font-icon-right-open btn--icon-large action action-detail" href="$Link"><span class="sr-only">View</span></a>
```

Comment:
I would suggest to just add back the `view-link` class for now. :)

```php
`$viewLink = $parser->getBySelector('.ss-gridfield-items .first .view-link');`
```

Reply:
Done